test/users/engin/compiler/parentheseslessRecConst.future

```diff
@@ -0,0 +1,6 @@
+bug: paranthesesless inner* constructor doesn't compile with --verify
```

Comment:
Spell check on "parenthesesless".  Parentheses are spelled with only one 'a', not two.